run.sh

```diff
@@ -71,10 +86,12 @@ solana-genesis \
   --bootstrap-storage-pubkey "$dataDir"/leader-storage-account-keypair.json \
   --ledger "$ledgerDir" \
   --dev
+tar jcfS "$ledgerDir/genesis.tar.bz2" -C "$ledgerDir" genesis.bin rocksdb
 
 abort() {
   set +e
   kill "$drone" "$validator"
+  wait $validator
```

Comment:
Specifically needed for `run-sanity.sh`; but I think generally propagating the exit status of validator is more useful than that of `kill` in this case.